etc/notebook/JsMVA/js/JsMVA.min.js

```diff
@@ -1 +1 @@
ttps://root.cern.ch/js/notebook/scripts/";var r="";if(requirejs.s.contexts.hasOwnProperty("_")){r=requirejs.s.contexts._.config.paths["JsMVA"].replace("JsMVA.min","")}if(console!==undefined&&typeof console.log=="function"){if(r!=""){console.log("JsMVA source dir:"+r.substring(0,r.length-1))}else{console.log("JsMVA source dir can't be resolved, requireJS doesn't have context '_', this will be a problem!")}}require.config({paths:{d3:e+"d3.min",JsRootCore:e+"JSRootCore.min",nn:r+"NeuralNetwork.min",dtree:r+"DecisionTree.min",NetworkDesigner:r+"NetworkDesigner.min"}});define(["JsRootCore"],function(e){return t({},e)})})(function(t,e){t.drawTH2=function(t,r){var i=e.parse(r);e.draw(t,i,"colz;PAL50;text")};t.drawDNNMap=function(t,r){var i=e.parse(r);e.draw(t,i,"colz;PAL50")};t.draw=function(t,r){var i=e.parse(r);e.draw(t,i)};t.drawNeuralNetwork=function(t,e){var r=JSON.parse(e);require(["nn"],function(e){e.draw(t,r)})};t.drawDecisionTree=function(t,e){require(["dtree"],function(r){var i=JSON.parse(e);r.draw(t,i)})};var r=function(t,r){require(["d3"],function(i){var n=i.select("#"+t+">.interactivePlot_Labels")[0][0];if(n!=null)return;var o=i.select("#"+t).style("position","relative");var a=o.append("svg").attr("class","interactivePlot_Labels").attr("width","200px").attr("height","50px").style({position:"absolute",top:"8px",right:"8px"});var s={pos:{x:150,y:0},rect:{width:10,height:10},dy:20,padding:10};canvas={width:160,height:70};var d=a.append("g").attr("id","legend");d.selectAll("g").data(r.fGraphs.arr).enter().append("g").each(function(t,r){var n=i.select(this);n.append("rect").attr("x",canvas.width-s.pos.x).attr("y",s.pos.y+r*s.dy).attr("width",s.rect.width).attr("height",s.rect.height).style("fill",function(t){return e.Painter.root_colors[t.fFillColor]});n.append("text").attr("x",canvas.width-s.pos.x+s.rect.width+s.padding).attr("y",s.pos.y+r*s.dy+s.rect.height).text(function(t){return t.fTitle}).style("fill",function(t){return e.Painter.root_colors[t.fFillColor]})});o.append("svg").attr("width","55px").attr("height","20px").style({position:"absolute",bottom:"15px",right:"40px"}).append("text").attr("x","5px").attr("y","15px").text(r.fGraphs.arr[0].fTitle.indexOf("Error on training set")!=-1?"Epoch":"#tree").style({"font-size":"16px"})})};t.drawTrainingTestingErrors=function(t,i){var n=e.parse(i);e.draw(t,n);r(t,n)};t.updateTrainingTestingErrors=function(t,i){var n=e.parse(i);e.redraw(t,n);r(t,n)};t.NetworkDesigner=function(t,e){require(["NetworkDesigner"],function(e){e.draw(t)})};t.outputShowCorrelationMatrix=function(t){require(["jquery","jquery-ui"],function(r){var i=e.parse(r("#"+t).html());if(!r("#dialog_"+t).length||r("#dialog_"+t).length<1){r("#"+t).parent().append("<div id='dialog_"+t+"' title='"+i.fTitle+"' style='width: 600px; height: 340px; z-index: 99;'></div>");e.draw("dialog_"+t,i,"colz;PAL50;text")}r("#dialog_"+t).dialog({autoOpen:true,width:600,show:{effect:"blind",duration:1e3},hide:{effect:"explode",duration:500}})})};return t});
```

Comment:
The only change here is `require.config({paths:{d3:e+"d3.v3.min" ...` to `equire.config({paths:{d3:e+"d3.min"`.

Reply:
I understand this makes it work again in a notebook? Did you try locally in your computer with a local Jupyter server?

Reply:
I tried, and verifed, it with modifying the request that my local client does to the server (using the chrome plugin Tamper I changed the request from d3.v3.min.js to d3.min.js).

However, I can not test this using my local environment as jsMVA.min.js is fetched from `https://root.cern.ch/js/jsmva/latest/js`, not my local root bin/src dir. (I also tried changing pyroot to fetch jsmva from a local webserver, but I was not able to make this work).

Reply:
To be completely sure this fixes it, perhaps @bellenot can replace `jsMVA.min.js` by your fixed version on root.cern.ch. Anyway, now it is broken.

Reply:
Sorry, but I don't see where d3 was changed from d3.v3.min.js to d3.min.js...

Reply:
It should suffice to replace any occurrence of "d3.v3.min" with "d3.min" in the file on the web server.

Reply:
I just want to understand what has changed and where. I can change whatever you want in jsmva (not in jsroot), but I want to know who is in charge of those files...

Reply:
OK, I created a `jsmva/1.1` directory with the required changes, and the `jsmva/latest` symlink is now pointing to it. Please let me know if it solves the issue.

Reply:
Thanks @bellenot!

From my testing it seems to work now. Waiting to hear back from the reporter here: https://root-forum.cern.ch/t/tmva-not-showing-plots-on-swan/29762/6